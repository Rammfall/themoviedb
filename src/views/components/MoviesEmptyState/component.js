import React from 'react'
import { Empty } from 'antd'
import { useIntl } from 'react-intl'

const MoviesEmptyState = () => {
  const { formatMessage } = useIntl()

  return (
    <div className='top-margin'>
      <Empty
        description={formatMessage({ id: 'movies.empty' })}
        image={Empty.PRESENTED_IMAGE_SIMPLE}
      />
    </div>
  )
}

export default MoviesEmptyState
