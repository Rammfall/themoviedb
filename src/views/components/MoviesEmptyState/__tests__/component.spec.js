import React from 'react'
import { shallow } from 'enzyme'

import MoviesEmptyState from '../component'

describe('MoviesEmptyState', () => {
  it('matches snapshot', () => {
    const component = shallow(<MoviesEmptyState />)

    expect(component).toMatchSnapshot()
  })
})
