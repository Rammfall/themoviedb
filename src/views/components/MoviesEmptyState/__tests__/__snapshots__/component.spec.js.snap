// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesEmptyState matches snapshot 1`] = `
<div
  className="top-margin"
>
  <Empty
    description="No movies found"
    image={<Simple />}
  />
</div>
`;
